appinfo/info.xml

```diff
@@ -25,7 +25,7 @@
 	<category>security</category>
 	<bugs>https://github.com/ChristophWurst/twofactor_admin/issues</bugs>
 	<dependencies>
-		<php min-version="8.0" max-version="8.3"/>
+		<php min-version="8.1" max-version="8.3"/>
```

Comment:
As long as Nextcloud 28 stays supported this bump is not allowed. 28 works with 8.0+.

Reply:
Is it? https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule

But I'll drop the two last commits and let the repo maintainers handle this.

Reply:
28 is EOL, but the main branch of this app is still compatible with it. The next release would also be offered to those outdated installations of 28 and if there is any 8.0 incompatibility it could break them.